### ISSUE TITLE
Main branch pauses videos when app background 

### ISSUE THREAD

**Issue by kalismeras61** [media-kit/media-kit#970]
Yesterday i used main branch and now i am seeing videos pauses when app in background. 

Is it related to latest changes ?

**Comment by alexmercerind** [comment]
Hi @kalismeras61,

The videos were always meant to pause in background on Android & iOS. We have an option to configure it:

https://github.com/media-kit/media-kit/blob/88c49c6c0f6187a131a4053f0f1a3a42f2922a8a/media_kit_video/lib/src/video/video_texture.dart#L129-L130

I can think further if you specify your platform.

**Comment by kalismeras61** [comment]
I was testing IOS it was working fine before but yesterday i wanted to use main branch but seems it is not working. 

 this.pauseUponEnteringBackgroundMode = false 
 this.resumeUponEnteringForegroundMode = true, 

Above one is correct for working background and still keeps playing on foreground?

**Comment by alexmercerind** [comment]
Yes. It should work.

**Comment by alexmercerind** [comment]
Do share any update.

**Comment by kalismeras61** [comment]
> Do share any update.

Tested an seems not working!. Now reverted back to pub.dev version

**Comment by t3chno** [comment]
Yes version 1.2.0 has background remote playback controller problem on iOS. 

Reproduce; 
- open notification center and now playing works fine
- minimize the app and maximize and - open notification center and now playing stops working.
